redskyctl/internal/commander/commander.go

```diff
@@ -173,55 +171,28 @@ func AddPreRunE(cmd *cobra.Command, preRunE func(*cobra.Command, []string) error
 	}
 }
 
-// ExitOnError converts all the error returning run functions to non-error implementations that immediately exit
-func ExitOnError(cmd *cobra.Command) {
-	// Convert a RunE to a Run
-	wrapE := func(runE func(*cobra.Command, []string) error) func(*cobra.Command, []string) {
-		return func(cmd *cobra.Command, args []string) {
-			err := runE(cmd, args)
-			if err == nil {
-				return
+// MapErrors wraps all of the error returning functions on the supplied command (and it's sub-commands) so that
+// pass any errors through the mapping function.
```

Comment:
nit, comment wording is a little broken, maybe missing `they` at the end of L174?